Observability.md

```diff
@@ -0,0 +1,30 @@
+# Observability
+
+[Observability](https://lightstep.com/blog/opentelemetry-101-what-is-observability/) generally refers to the ability to understand an application’s performance based on output data. This telemetry data is divided into three types: metrics, tracing and logging.
+
+Microsoft Graph SDKs should provide the ability for application developers to gather data about the calls made via the SDKs. No telemetry data is captured and transmitted to Microsoft beyond what is included in the HTTP request headers.
```

Comment:
Should the SDK assume Observability will always be used (Basically creating collectors, exporters and spans in all the methods we believe are relevant) and let developers leverage them (or not)?

Reply:
It is up to the app developer to choose their collectors/exporters. By default we assume none are configured.  Our job us just to create spans.